### PR TITLE
Scipy talk

### PR DESCRIPTION
**Pull request by LeJit**
Added Instructor Notebook.

TODO:
- Remove old notebook
- Create Student's Notebook
- Update Colab link to the Student Notebook.

### CODE REVIEW

File: 2021_scipy_tutorial/datasheet.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Motivation
```
Comment (MiroDudik) [inline 2021_scipy_tutorial/datasheet.md:1]: remove this file

File: 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb [hunk 2]
```diff
@@ -0,0 +1,4620 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "EAtkvVOEZqMb"
+      },
+      "outputs": [],
+      "source": [
+        "# Compute the balanced error rate and false negative rate difference for each of the predictors on the test data.\n",
+        "weighted_error_exercise = [(1 - balanced_accuracy_score(Y_test, pred.predict(X_test))) for pred in predictors_exercise]\n",
```
Comment (MiroDudik) [inline 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb:3989]: `weighted_error_exercise` -> `balanced_error_exercise`
[there are 4 instances here / in collab]

File: 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb [hunk 3]
```diff
@@ -0,0 +1,4620 @@
...
+      "metadata": {
+        "id": "YscNZsYU1rCY"
+      },
+      "outputs": [],
+      "source": [
+        "# Record and evaluate the output of the trained ThresholdOptimizer on test data\n",
+        "\n",
+        "Y_pred_postprocess = postprocess_est.predict(X_test, sensitive_features=A_test)\n",
+        "metricframe_postprocess = MetricFrame(\n",
+        "    metrics=metrics_with_err_bars,\n",
```
Comment (MiroDudik) [inline 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb:2976]: Now that the error bars exercise is optional, should we just have a separate dictionary called `metrics` or `tutorial_metrics` that would just contain all of the metrics that are included in `metrics_to_display`? Then we wouldn't need to use `metrics_to_display` after the error bars section (we would still need to use differences_to_display).

Alternatively, we can keep things as is and I can mention these variables when I go through the MetricFrame section.

**Review by MiroDudik — CHANGES_REQUESTED** [review]
Besides the mentioned issues (and once we figure what to do re. `metrics_with_err_bars`, also add the student version where exercise cells are presented with the dunder version.

File: 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb (hunk 3 above)
Comment (LeJit) [reply 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb:2976]: Honestly, we could just use `metric_dict` in the later sections. Showing `count` isn't super useful in the later sections, so we only need to visualize performance across the three metrics.

**Comment by LeJit** [comment]
@MiroDudik I updated the instructor notebook to use `mf2` as `metricframe_unmitigated` and for later sections to use `metrics_dict`. This allows us to remove the `metrics_to_display` indexing.

File: 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb (hunk 3 above)
Comment (MiroDudik) [reply 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb]: great. can you implement?

Comment (MiroDudik) [reply 2021_scipy_tutorial/fairness-in-AI-systems-instructors.ipynb]: [i see below that you just did]

**Comment by MiroDudik** [comment]
@LeJit : this is almost there. Two more changes: update `requirements.txt` so the fairlearn version is 0.7.0, and remove the old `.ipynb` file (I have already fixed the link from `README.md`).

**Pull request merged**